pkg/webhook/loadbalancer/validator.go

```diff
@@ -125,6 +129,11 @@ func checkListeners(lb *lbv1.LoadBalancer) error {
 }
 
 func checkHealthyCheck(lb *lbv1.LoadBalancer) error {
+	// The healthyCheck related configuration is only valid for VM type LB.
+	if lb.Spec.WorkloadType == lbv1.Cluster {
+		return nil
+	}
```

Comment:
Shouldn't the cluster-type LBs with health check defined be rejected? Otherwise the controller will work on those LB objects.

Reply:
The controller works on HealthCheck on function `ensureVMLoadBalancer`, which is upon VM type LB.

And, if some legacy LBs happen to have such fields, skip them is fine.

Reply:
Thanks for the explanation. It makes much more sense now.